test/test_scanner.py

```diff
@@ -282,6 +282,16 @@ def test_gnutls_2_3_11(self):
                     ],
                 )
 
+    @unittest.skipUnless(os.getenv("LONG_TESTS") == "1", "Skipping long tests")
+    def test_gnutls_rpm_2_12_23(self):
+        """ Test detection of gnutls 3.6.11 """
```

Comment:
Please update the docstring to match the test (currently it's looking for 2.12.23, not 3.6.11)